test/scUSDC.t.sol

```diff
@@ -245,7 +245,7 @@ contract scUSDCTest is Test {
         vm.prank(keeper);
         vault.rebalance();
 
-        assertApproxEqRel(vault.getLtv(), vault.targetLtv(), 0.001e18);
+        assertApproxEqRel(vault.getLtv(), vault.targetLtv(), 0.005e18);
```

Comment:
what's going on here? why is it less precise? isn't the point of the PR to make it more precise?